### PR TITLE
OKTA-438759 Convert configure-access-policy to single page guide

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Convert "Configure an access policy" (/configure-access-policy/) to single page guide
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-438759](https://oktainc.atlassian.net/browse/OKTA-438759)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* An OIDC client app for testing access policies
+
+<ApiAmProdWarning />
+
+---
+
+## Overview
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md:28]: It's not in the Confluence doc, but I think that Christine prefers that we avoid using "Overview" here. Something like "About configuring access policies" could be more informative.

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

Just one question.

File: packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+---
+
+This guide helps you understand access policies and how to configure them for common scenarios.
+
+---
+<!-- Nutrition facts bullets -->
+**Learning outcomes**
+
+* Understand access policies.
+* Configure an access policy to limit which scopes some clients can access.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md:16]:
```suggestion
* Configure an access policy to limit the scopes that some clients can access.
```

Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md:16]:
```suggestion
* Configure an access policy to limit which scopes that some clients can access.
```

File: packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+5. Select **The following clients:** and start typing the names of the Okta OpenID Connect applications that you want to cover with the access policy. This field automatically displays a list of applications that match what you type.
+
+6. Click **Create Policy**.
+
+### Create the rule
+
+1. Select the name of the access policy that you just created, and then select **Add Rule**.
+
+2. Enter a **Name** for the rule, and then for this example leave the defaults for **IF Grant type is** and **AND User is**.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md:67]:
```suggestion
2. Enter a **Name** for the rule, and then for this example, leave the defaults for **IF Grant type is** and **AND User is**.
```

File: packages/@okta/vuepress-site/docs/guides/refresh-tokens/main/index.md [hunk 4]
```diff
@@ -97,7 +97,7 @@ See [Refresh token object](/docs/reference/api/apps/#refresh-token-object).
 
 ### Refresh token lifetime
 
-Refresh token lifetimes are managed through the [Authorization Server access policy](/docs/guides/configure-access-policy/overview/). The default value for the refresh token lifetime (`refreshTokenLifetimeMinutes`) for an [Authorization Server access policy](/docs/reference/api/authorization-servers/#actions-object) is **Unlimited**, but expires every seven days if hasn't been used. When you use a refresh token with a SPA, make sure that you keep a short refresh token lifetime for better security.
+Refresh token lifetimes are managed through the [Authorization Server access policy](/docs/guides/configure-access-policy/). The default value for the refresh token lifetime (`refreshTokenLifetimeMinutes`) for an [Authorization Server actions object](/docs/reference/api/authorization-servers/#actions-object) is **Unlimited**, but expires every seven days if hasn't been used. When you use a refresh token with a SPA, make sure that you keep a short refresh token lifetime for better security.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/refresh-tokens/main/index.md:100]:
```suggestion
Refresh token lifetimes are managed through the [Authorization Server access policy](/docs/guides/configure-access-policy/). The default value for the refresh token lifetime (`refreshTokenLifetimeMinutes`) for an [Authorization Server actions object](/docs/reference/api/authorization-servers/#actions-object) is **Unlimited**, but expires every seven days if it hasn't been used. When you use a refresh token with a SPA, make sure that you keep a short refresh token lifetime for better security.
```

**Review by christinechoy-okta — APPROVED** [review]
Just a few nits, otherwise LGTM!

**Pull request merged**